### PR TITLE
Add recipe for scattertext

### PR DESCRIPTION
**Pull request by synapticarbors**
https://twitter.com/jasonkessler/status/896885611428102144

This depends on https://github.com/conda-forge/staged-recipes/pull/3576, so will fail until that PR is merged. 

@JasonKessler -- are you interested in being a co-maintainer of this conda recipe?

### CODE REVIEW

File: recipes/scattertext/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+  skip: True  # [win32]
+  entry_points:
+    - scattertext = scattertext.CLI:main
+
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - nose >=1.0
```
Comment (mwcraig) [inline recipes/scattertext/meta.yaml]: Is nose required for install?

Comment (synapticarbors) [inline recipes/scattertext/meta.yaml]: Yes: 

https://github.com/JasonKessler/scattertext/blob/master/setup.py#L31

Also, I'm still hoping to get a response from @JasonKessler about co-maintaining this recipe. Unfortunately the asshole that organized the Charlottesville march shares the same name and I'm guessing our good data scientist friend might have missed the original mention during all that.

Comment (JasonKessler) [inline recipes/scattertext/meta.yaml]: Sorry I've been MIA!  

Nose isn't required to run Scattertext, and I'm about to push out an update that doesn't have it in "install_requires" or "setup_requires" but retains it in "tests_require".

Thanks a lot for creating this recipe.

Comment (synapticarbors) [inline recipes/scattertext/meta.yaml]: @JasonKessler -- Great, I'll update the recipe. Are you interested in being a co-maintainer of this recipe? It just means that you'll have write permission to the feedstock for `scattertext` and can push updates there (although I'm happy to keep the conda-forge recipe up-to-date).

Comment (JasonKessler) [inline recipes/scattertext/meta.yaml]: That could be useful, please add me.  And thanks for keeping the conda-forge recipe up-to-date.

**Comment by synapticarbors** [comment]
@mwcraig - Ready to merge

**Pull request merged**